utils/denoDoc.ts

```diff
@@ -0,0 +1,363 @@
+import { exec, OutputMode } from "https://deno.land/x/exec@0.0.5/mod.ts";
+import { JSONSchema7 } from "json-schema";
+
+/**
+ * Transforms myPropName into "My Prop Name" for cases
+ * when there's no label specified
+ *
+ * TODO: Support i18n in the future
+ */
+const beautifyPropName = (propName: string) => {
+  return (
+    propName
+      // insert a space before all caps
+      .replace(/([A-Z])/g, " $1")
+      // uppercase the first character
+      .replace(/^./, function (str) {
+        return str.toUpperCase();
+      })
+  );
+};
+
+/**
+ * We use $id inside of schemas to match components and functions.
+ *
+ * This function is responsible for generating this $id from the
+ * type's import URL
+ *
+ * E.g: importUrl: file:///Users/lucis/deco/live/std/commerce/types/Product.ts
+ */
+const get$idFromImportUrl = (importUrl: string) => {
+  // TODO: Improve this to not include live's version when using deno.lan
+  // E.g: live@0.1.16-functions/std/commerce/types/ProductList.ts
+  return "live" + importUrl?.split("/live")[1];
+};
+
+export async function readDenoDocJson(filePath: string) {
+  const { output: rawOutput } = await exec(`deno doc ${filePath} --json`, {
+    output: OutputMode.Capture,
+  });
+  const denoDocOutput = JSON.parse(rawOutput);
+  return denoDocOutput;
+}
+
+export function getFunctionOutputSchemaFromDocs(
+  docs: DenoDocResponse[],
+  entityName: string
+) {
+  const defaultFunctionExport = docs.find(
+    ({ name, kind }) => name === "default" && kind === "variable"
+  );
```

Comment:
This works for all this kind of exports default?
```ts
// Default exports
export default expression;
export default function functionName() { /* … */ }
export default class ClassName { /* … */ }
export default function* generatorFunctionName() { /* … */ }
export default function () { /* … */ }
export default class { /* … */ }
export default function* () { /* … */ }
```
Get these exports from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
No, we won't support every possible use case in the beginning. 

Since it's an intrinsically sensible translation, we'll have a good documentation and evolve with feedback